### DISCUSSION TITLE
Making it your own

### DISCUSSION THREAD

**Discussion by Xavier509** [UseInterstellar/Interstellar#911] [Q&A] [answered]
Am I allowed to use parts of the code or just change some of it to make my own unblocked proxy? Please let me know as soon as you can because I just need a kickstart to my own proxy because of problems I am having trouble with my code.

**Comment by lakshinhemachandran** [comment]
I'm pretty sure you are not allowed to reproduce, copy, or distribute, according to the license.
But you can with explicit permission from Xbubbo.

**Reply by Xavier509** [reply]
Is xbubbo a person who checks the discussion often and can grant me permission? Remember I only need a little bit to kickstart mine.

**Reply by lakshinhemachandran** [reply, marked as answer]
I'm not sure if Xbubbo will check  _discussions_ often, but he usually checks issues. But this isn't really an issue isn't it?
Also, remember I just starting following this project a few weeks ago, so I might not know much. All I know for sure is that you need permission from _someone_ to get their code.
